src/draive/generation/text/types.py

```
@@ -14,10 +14,10 @@ class TextGenerating(Protocol):
     async def __call__(
         self,
         *,
-        instruction: Instruction | str | None,
-        input: Prompt | Multimodal,  # noqa: A002
-        tools: Toolbox | Iterable[Tool] | None,
-        examples: Iterable[tuple[Multimodal, str]] | None,
+        instruction: Instruction | None,
+        input: Prompt | MultimodalContent,  # noqa: A002
```

Pylint (3.3.7)

[warning] Redefining built-in 'input'

(W0622)